plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -2641,3 +2641,118 @@ func TestPodToleratesTaints(t *testing.T) {
 		}
 	}
 }
+
+func makeNonBestEffortResources(milliCPU int64, memory int64) api.ResourceList {
```

Comment:
The predicate_test.go defines ``makeAllocatableResources`` function which I can use. I don't know why I did not use that before.